src/public/publicAPIs.ts

```diff
@@ -171,6 +179,26 @@ export function print(): void {
   window.print();
 }
 
+/**
+ * This is a sample API which demonstrates the usage of IsAPISupportedByPlatform API
+ * TODO: Remove this API before check-in
+ * @param callback Callback to invoke when API processing is done
+ */
+export function sampleAPI(callback: (response: BoolResponse) => void): void {
```

Comment:
I think what we discussed is to have an error code as the first parameter of the callback and any response objects follows.
e.g.
```
callback: (error: ErrorCode, files:   File[])
```

Reply:
I demoed by putting error inside the response interface itself. But, the way you have showed it, that's also equivalent and I have no concerns with that. Anything is fine. 

BTW, this is just sample code and I will remove this API before check-in.

Reply:
I remember in the design review, we mentioned that callback will have (error, results) format. The reason for that is for the caller to not know about what result is expected by the callback. For version usecase, where you need to call the callback only with the error, it is much easier to call the method with one error argument than to create a definite type of object and plug in an error there.

Reply:
OK. Let's use the format 

callback: (error: SdkError, result:   File[])